projects/core/directives/textfield-controller/textfield-autocomplete.directive.ts

```diff
@@ -17,7 +17,7 @@ export const TUI_TEXTFIELD_AUTOCOMPLETE =
 })
 export class TuiTextfieldAutocompleteDirective extends TuiController {
     @Input('tuiTextfieldAutocomplete')
-    autocomplete: TuiAutofillFieldName | '' = '';
+    autocomplete: TuiAutofillFieldName | '' | null = '';
```

Comment:
I don't see why `null` is here, but this directive would be removed in 3.0 anyway.

Reply:
fixed